pages/goals/index.tsx

```diff
@@ -83,7 +81,7 @@ export default function Goals(props) {
               <GoalsSectionComponent
                 userGoals={section.userGoals}
                 sectionName={section.sectionName}
-                componentUsageType={GoalsComponentSection.GOALS}
+                inProfile={false}
```

Comment:
over here, you could write !inProfile for shorthand to pass the value as false into the component